migration/migrator/migrations/course/20210113174909_registration_subsection.py

```diff
@@ -0,0 +1,7 @@
+
+def up(config, database, semester, course):
+    database.execute('ALTER TABLE users ADD COLUMN registration_subsection character varying(255)')
```

Comment:
Make this an 'add if does not exist' 
(see other migration files for exact syntax)